Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN make install
 
 # Fetch umeed binary
 FROM base-builder AS umeed-builder
-ARG UMEE_VERSION=v2.0.0
+ARG UMEE_VERSION=robert/cosmos-046
```

Comment:
we need to add a task to revert this once we will merge in `umee` repo.

Reply:
#344